test/Tests.Infrastructure/InterversionTestBase.cs

```diff
@@ -97,7 +97,17 @@ public abstract class InterversionTestBase : RavenTestBase
             var testServerPath = NewDataPath(prefix: serverVersion);
             CopyFilesRecursively(new DirectoryInfo(serverPath), new DirectoryInfo(testServerPath));
 
-            var locator = new ConfigurableRavenServerLocator(testServerPath);
+            bool includeLibuvArg = true;
+            var version = serverVersion.Split(".");
+            var highestVersionWithoutLibuv = new int[] { 4, 1, 4 };
```

Comment:
Not needed. This should be supported in every 4.0.7+ version. Just make sure you are using the right one.